backend/package.json

```diff
@@ -109,6 +109,23 @@
 					"description": "The verbosity of logging according to the following order: trace > debug > info > warn > error > fatal > off.",
 					"scope": "resource"
 				},
+				"ApplicationWizard.TargetFolder": {
+					"type": "string",
+					"default": "",
```

Comment:
since no browse button in settings, I'm afraid that user wont know what value to place here.

![image](https://user-images.githubusercontent.com/57438361/105679602-5fac6a00-5ef7-11eb-8805-30bb556d2af1.png)

Reply:
but in bas the default is /home/... and in vscode the default is c:/user... so this is the only option that we won't have to override this value

Reply:
My comment is for PM. The user can mistake in the value he places here since no browse button,